examples/examples_py_test.go

```diff
@@ -45,6 +45,7 @@ func TestAwsPy(t *testing.T) {
 }
 
 func TestAzurePy(t *testing.T) {
+	t.Skip("Skipping test due to updates in Image resource")
```

Comment:
Skips still needed?

Reply:
Not that particular one, but see below skip condition due to AZURE_LOCATION. Removing L48.